nwbwidgets/dashboard.py

```diff
@@ -0,0 +1,89 @@
+import ipywidgets as widgets
+from pathlib import Path
+
+from pynwb import NWBHDF5IO
+from nwbwidgets import nwb2widget
+from dandi.dandiapi import DandiAPIClient
+
+
+def dashboard():
```

Comment:
can we call this something else? Like a file selector?

Reply:
just renamed to `input_form`, hows that? @bendichter